elements/pfe-card/src/pfe-card.scss

```diff
@@ -52,7 +51,7 @@ $variables: (
   flex-direction: column;
   justify-items: flex-start;
   // This allows the card to fill it's container if necessary
-  align-self: stretch;
+  align-self: #{pfe-local($cssvar: AlignSelf, $fallback: stretch)};
```

Comment:
Stretch is the default but in grid layouts where you don't want the card to fill the container, this can be set to flex-start